### PR TITLE
Update PF4 dependencies to v4

### PR DESCRIPTION
**Pull request by Hyperkid123**
### Dependencies
- [x] platform (https://github.com/RedHatInsights/insights-chrome/pull/817)
- [x] frontend components (beta releases are avaiable) (https://github.com/RedHatInsights/frontend-components/issues/561)
- [x] data driven forms (https://github.com/data-driven-forms/react-forms/pull/549)
- [x] Survey editor (https://github.com/data-driven-forms/form-builder/pull/23)

### Changes
- update to v4 release of PF4
- temporarily disabled components from third-party libraries that depend on older version of PF
  - primary toolbar (fce)
  - table toolbar (fcd)
  - survey editor (also contains toolbar in the dual list selector)
- we are missing the base css file which is distributed by the chrome app
 

### Current progress without platform support.
Once the platform supplies the styling, the migration should not take long. Its mostly a matter of updating tests and some layouts. (ignore the broken chrome)

screenshot-ci foo redhat com_1337-2020 06 08-15_58_26

### CODE REVIEW

**Comment by Hyperkid123** [comment]
Most of the styling is fixed, now to upgrade the libraries using PF

**Comment by Hyperkid123** [comment]
Beta builds of DDF and FCE dependencies are available and added. There are no more runtime crashes because of them and the migration is almost ready.

**Comment by Hyperkid123** [comment]
Ok, we are set. Now we need to just wait for the chrome to update or the deadline. Whatever comes first.

**Comment by Hyperkid123** [comment]
@gmcculloug the update is ready and the chrome changes were merged (not deployed yet).

This PR can be merged whenever we need it.

**Comment by Hyperkid123** [comment]
cc @eclarizio

File: src/presentational-components/shared/loader-placeholders.js [hunk 1]
```diff
@@ -77,22 +78,24 @@ const SkeletonContainer = styled.div`
...
-                <Skeleton height={70} width="85%" className="pf-u-mb-lg" />
-                <Skeleton height={5} width="90%" className="pf-u-mb-sm" />
-                <Skeleton height={5} width="100%" className="pf-u-mb-sm" />
-                <Skeleton height={5} width="76%" className="pf-u-mb-sm" />
-              </CardBody>
-            </Card>
-          </GalleryItem>
-        ))}
-      </Gallery>
+  <Grid>
```
Comment (eclarizio) [inline src/presentational-components/shared/loader-placeholders.js:81]: Does this not need the `hasGutter` because the old one had `gutter="md"`?

Comment (Hyperkid123) [reply src/presentational-components/shared/loader-placeholders.js:81]: it does!

**Pull request merged**